internal/business/server/openapi.go

```diff
@@ -148,6 +148,70 @@ func (s *openAPIServer) Callback(ctx context.Context, req openapi.CallbackReques
 	}, nil
 }
 
+// Logout implements openapi.StrictServerInterface.
+func (s *openAPIServer) Logout(ctx context.Context, request openapi.LogoutRequestObject) (openapi.LogoutResponseObject, error) {
+	slogctx.Debug(ctx, "Logout() called")
+	defer slogctx.Debug(ctx, "Callback() completed")
+
+	rw, err := middleware.ResponseWriterFromContext(ctx)
+	if err != nil {
+		slogctx.Error(ctx, "Failed to get response writer from context", "error", err)
+
+		body, status := s.toErrorModel(serviceerr.ErrUnknown)
+		return openapi.LogoutdefaultJSONResponse{
+			Body:       body,
+			StatusCode: status,
+		}, nil
+	}
+
+	sessionID := request.Params.Cookie
```

Comment:
I think this will contain all cookies, so we have to parse this first ... maybe like this:
```go
    header := http.Header{}
    header.Add("Cookie", request.Params.Cookie)
    httpReq := &http.Request{Header: header}
    for _, cookie := range httpReq.Cookies() {
        // filter the session cookie and trigger the logout for it
    }
```

This also means that instead of using `MakeSessionCookie` and `MakeCSRFCookie` below, we can simply collect the related cookies in the loop (session cookie and CSRF cookie), set their `Value` to `""` and their `MaxAge` to `-1` and call `http.SetCookie(rw, cookie)` on them.